# SafetyMsg.msg
Header header
float32 safety_rate
float32 epsilon